pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "j5"
+version = "0.0.2"
+description = "J5 Robotics API"
+authors = ["Dan Trickey <contact@trickey.io>"]
```

Comment:
We might as well update this to j5 contributors to match the licence

Reply:
Done